src/main/java/com/agricraft/agricore/lang/AgriString.java

```diff
@@ -15,20 +15,17 @@
     public AgriString() {
         this.normal = "add translations in here.";
         this.translations = new HashMap<>();
-        this.translations.put("en_US", normal);
     }
 
     public AgriString(String value) {
         this.normal = value;
         this.translations = new HashMap<>();
-        this.translations.put("en_US", normal);
     }
 
     public AgriString(String normal, Map<String, String> translations) {
         this.normal = normal;
         this.translations = translations;
         if (this.translations.isEmpty()) {
-            this.translations.put("en_US", normal);
         }
```

Comment:
I did not like this to be honest, firstly, if I understand correctly, if the translation fails, it will return this either way, secondly, Minecraft has changed the language keys to be all lower case as well, and I do not feel like updating all jsons whenever mc changes the keys.

Furthermore, I am considering dropping this localization all together, since it can be perfectly done with resource packs as well.

Reply:
How do the resource pack translations work? Do you just replace all the strings with keys?